src/process/os/windows.rs

```diff
@@ -25,7 +32,63 @@ pub trait ProcessExt {
 
 impl ProcessExt for Process {
 	fn environ(&self) -> ProcessResult<HashMap<String, String>> {
-		todo!()
+		let mut delay = 1;
```

Comment:
This section looks like it could use some comments explaining the code including the magic numbers.